synapse/cortex.py

```diff
@@ -1297,13 +1301,18 @@ def getView(self, iden=None):
     async def _layrFromNode(self, node):
 
         info = await node.dict()
+        ltyp = info.get('type')
 
-        ctor = self.layrctors.get(info.get('type'))
+        ctor = self.layrctors.get(ltyp)
         if ctor is None:
-            logger.warning('layer has invalid type: %r %r' % (node.name(), info.get('type')))
+            logger.warning('layer has invalid type: %r %r' % (node.name(), ltyp))
             return None
 
-        layr = await ctor.anit(self, node)
+        if ltyp == 'lmdb' and self.conf.get('nomnommem'):
```

Comment:
if we pushed the dedicated value from the config into the node; each layer implementation could do its own thing for what "dedicated" means. This would keep a consistent calling convention here.